packages/user/src/User/User.purs

```diff
@@ -758,3 +764,28 @@ callBottega f = do
 
 getPackages :: Aff (Array Package)
 getPackages = Bottega.getPackages
+
+getPaywallOpenings :: Aff (Array PaywallOpening)
+getPaywallOpenings =
+  Persona.getPaywallOpenings =<< requireToken
+
+openPaywall :: Int -> Int -> Int -> Array String -> Aff Unit
+openPaywall days hours minutes products = do
```

Comment:
You could use `onlyToProducts` as variable name instead of `products` and then you could use the `{onlyToProducts}` syntax on line 785.